cmake/setup_KWIVER.sh.in

```diff
@@ -3,4 +3,4 @@
 # https://github.com/Kitware/seal-tk/blob/master/LICENSE for details.
 
 export KWIVER_PLUGIN_PATH=\
-"@PROJECT_BINARY_DIR@/lib/kwiver/modules:$KWIVER_PLUGIN_PATH"
+"@sealtk_BUILD_LIBDIR@/kwiver/modules${KWIVER_PLUGIN_PATH:+:$KWIVER_PLUGIN_PATH}"
```

Comment:
`ctest -V -R SourceCheck` indicates that this line is too long. Please fix.

Reply:
Done